src/constants/status.js

```diff
@@ -51,7 +51,7 @@ export const statusMapper = {
   },
   error: {
     text: 'Error',
-    Icon: iconMapper.timesCircle,
+    Icon: iconMapper.exclmationCircle,
```

Comment:
This would effect other error status's that may be used across other views. If we need to have a different one. Add another entry. For example `errorWithExclamationCircle` :smile: